scripts/webframeworks-deploy-tests/nextjs/app/app/api/static/route.ts

```diff
@@ -1,3 +1,5 @@
+export const dynamic = 'force-static'
```

Comment:
Next 15 does not cache route handlers by default, we need to add `force-static` when we want to cache the route handler.